README.md

```diff
@@ -35,667 +35,45 @@ Currently k8s-cleaner has rich set of working examples to identify and list unus
 
-💪 **Reduced Resource Bloat**: Minimize resource bloat and maintain a clean and organized cluster, improving overall performance and stability.
+To add an example, simply create a new file in the example directory with a descriptive name and put your Cleaner configuration within the file. Once you've added your example, feel free to submit a pull request to share it with the community.
 
-By combining the flexibility of scheduling, the accuracy of label filtering, the power of Lua-based criteria, and the ability to remove or update stale resources, Cleaner empowers users to effectively manage their Kubernetes environments and optimize resource usage.
-
-## Deploying the K8s Cleaner
-To deploy the __k8s-cleaner__ to your Kubernetes cluster, run the following command:
-
-```
-kubectl apply -f https://raw.githubusercontent.com/gianlucam76/k8s-cleaner/main/manifest/manifest.yaml
-```
-
-## Removing Unwanted Secrets
-
-To remove all Secrets from the test namespace every day at 1 AM, use the following YAML configuration:
-
-```yaml
-apiVersion: apps.projectsveltos.io/v1alpha1
-kind: Cleaner
-metadata:
-  name: cleaner-sample
-spec:
-  schedule: "* 1 * * *" # Runs every day at 1 AM
-  resourcePolicySet:
-    resourceSelectors:
-    - namespace: test
-      kind: Secret
-      group: ""
-      version: v1
-  action: Delete # Deletes matching Secrets
-```
-
-This configuration instructs the Cleaner to scan the test namespace every day at 1 AM, identify all Secrets, and effectively eliminate them, ensuring a clean and organized cluster.
-
-## Selecting Resources with Label Filters
-
-__Cleaner__ can select resources based on their labels, enabling precise resource management. For instance, to eliminate Deployments in the __test__ namespace with both ``serving=api`` and ``environment!=production`` labels, follow this YAML configuration:
-
-```yaml
-apiVersion: apps.projectsveltos.io/v1alpha1
-kind: Cleaner
-metadata:
-  name: cleaner-sample1
-spec:
-  schedule: "* 0 * * *" # Executes every day at midnight
-  resourcePolicySet:
-    resourceSelectors:
-    - namespace: test
-      kind: Deployment
-      group: "apps"
-      version: v1
-      labelFilters:
-      - key: serving
-        operation: Equal
-        value: api # Identifies Deployments with "serving" label set to "api"
-      - key: environment
-        operation: Different
-        value: prouction # Identifies Deployments with "environment" label different from "production"
-  action: Delete # Deletes matching Deployments
-```
-
-By utilizing label filters, you can refine the scope of resource management, ensuring that only specific resources are targeted for removal or update. This targeted approach helps maintain a clean and organized Kubernetes environment without affecting unintended resources.
-
-## Namespace Selector
```

Comment:
We need a section in the documentation where we explain how to set resourcePolicySet.